lib/full-site-editing/default-template-types.php

```diff
@@ -15,71 +15,71 @@ function gutenberg_get_default_template_types() {
 	$default_template_types = array(
 		'index'          => array(
 			'title'       => _x( 'Index', 'Template name', 'gutenberg' ),
-			'description' => __( 'The default template which is used when no other template can be found', 'gutenberg' ),
+			'description' => __( 'The default template used when no other template is available. This is a required template in WordPress.', 'gutenberg' ),
 		),
 		'home'           => array(
 			'title'       => _x( 'Home', 'Template name', 'gutenberg' ),
-			'description' => __( 'The home page template, which is the front page by default. If you use a static front page this is the template for the page with the latest posts', 'gutenberg' ),
+			'description' => __( 'Template used for the main page that displays blog posts. This is the front page by default in WordPress. If a static front page is set, this is the template used for the page that contains the latest blog posts.', 'gutenberg' ),
 		),
 		'front-page'     => array(
 			'title'       => _x( 'Front Page', 'Template name', 'gutenberg' ),
-			'description' => __( 'Used when the site home page is queried', 'gutenberg' ),
+			'description' => __( 'Template used to render the front page of the site, whether it displays blog posts or a static page. The front page template takes precedence over the "Home" template.', 'gutenberg' ),
 		),
 		'singular'       => array(
 			'title'       => _x( 'Singular', 'Template name', 'gutenberg' ),
-			'description' => __( 'Used when a single entry is queried. This template will be overridden by the Single, Post, and Page templates where appropriate', 'gutenberg' ),
+			'description' => __( 'Template used for displaying single views of the content. This template is a fallback for the Single, Post, and Page templates, which take precedence when they exist.', 'gutenberg' ),
 		),
 		'single'         => array(
 			'title'       => _x( 'Single', 'Template name', 'gutenberg' ),
-			'description' => __( 'Used when a single entry that is not a Page is queried', 'gutenberg' ),
+			'description' => __( 'Template used to display a single post or entry.', 'gutenberg' ),
```

Comment:
Is the term "entry" valuable here? We might be able to do without that word entirely.

Reply:
It's to somehow differentiate from the single-post case.

Reply:
I think we should remove the single-post from the default set, actually.

Reply:
It is also important to clarify that the template can display post types other than `post`, which comes in to play whenever a CPT is registered.

In fact this is the reason that `single-post` was included as well. It's explicit nature means we can enable folks to intuitively add a template just for `post`s which will likely be a common use case. There is a dedicated template for Pages, it makes sense for there to be a dedicated template for Posts as well, imo.

Reply:
I don't think that would be very intuitive. Pages are a special case when it comes to templates in WP. I don't think I have seen major themes that use `single-post`. Almost always you want to capture as many single post type entities as possible with it, and discriminate pages, which are not chronological.

If a custom post type requires a special treatment, it makes sense for that to be created specifically.